src/02-common-principles.md

```diff
@@ -105,6 +105,12 @@ misunderstanding we clarify them here.
     The modality may overlap with, but should not be confused with
     the **data type**.
 
+1.  **Entity** - a portion of a file name, consisting of a **key** and corresponding
+    **value** separated by a hyphen. Supported entities are defined in
+    [Appendix IX](99-appendices/09-entities.md); further, whether they are allowed,
```

Comment:
I can't think of a case that is allowed but neither OPTIONAL or REQUIRED, so we should avoid introducing a term that implies a distinction.

```suggestion
    [Appendix IX](99-appendices/09-entities.md); further, whether they are
```